src/status_im/chat/models/message.cljs

```diff
@@ -39,9 +38,9 @@
   [{:keys [content content-type] :as message} chat-id current-chat?]
   (let [emoji? (message-content/emoji-only-content? content)]
     ;; TODO janherich: enable the animations again once we can do them more efficiently
-    (cond-> (assoc message :appearing? true)
-      (not current-chat?)
-      (assoc :appearing? false)
+    (cond-> message
+      current-chat?
+      (assoc :seen true)
```

Comment:
could you elaborate?

Reply:
`appearing` is useless, not needed.
If the message is received in the current chat it is considered seen. It was already the case before just not done here